docs/wiki/framework/vehicles-framework.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: wiki
+title: Vehicles Framework
+description: Explains how to set-up vehicles startup delay.
+group: framework
+order: 5
+parent: wiki
+mod: ace
+version:
+  major: 3
+  minor: 14
+  patch: 0
+---
+
+## 1. Engine startup delay
+
+The engine has to be started before the vehicle can move. Delay can be configured per class via config or per vehicle via script.
+By default starting the engine takes aprox. 1 to 2 seconds.
+
+### 1.1 Setting the startup delay by config
+
+```cpp
+class CfgVehicles {
+    class MyFuelTruck {
+        ace_vehicles_engineStartDelay = 3; // Startup delay in seconds
+    };
+    class MyCar {
+        ace_vehicles_engineStartDelay = 2.2;
+    };
+    class MyElectricCar {
+        ace_vehicles_engineStartDelay = 0.1;
+    };
+};
+```
+
+### 1.2 Setting the startup delay by script
+
+`ace_vehicles_fnc_setVehicleStartDelay`
+
```

Comment:
```suggestion
Has global effects, so can be called just on server.

```

Reply:
setVariable[.., .., true] will broadcast value from any PC. Isn't it?

Reply:
```suggestion
Has global effects.

```
Yeah any machine would be fine, I thought it might be good to mention because a lot of people will probably but it in the init field. although there is no real harm if the setVar is rerun lots of times